src/components/Dish/Dish.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+    <div class="dish-inner" :class="layout">
+        <div class="image-wrapper">
+            <img :src='image' alt="" />
+        </div>
+        <div class="dish-label">
+            <div class="dish-label-title">
+                <h3>{{ title }}</h3>
+                <p>{{ description | descriptionLimit }}</p>
+            </div>
+            <div class="dish-label-price">
+                <span>{{ pricetext }}</span>
+            </div>
+        </div>
+        <div class="dish-add-item"  @click="addDishItem(dish)" v-if="layout == 'horizontal'">
+            +
+        </div> 
+    </div>
+</template>
+<script>
+export default {
+    name: 'Dish',
+    data () {
+        return {
+            pricetext: ''
+        }
+    },
+    props: ['title', 'image', 'description', 'price', 'ingredients', 'layout'],
+    methods: {
+        addDishItem(id) {
+            // Do functionality here for adding items to cart or something else
+            console.log("Adding item " + id)
+        } 
+    },
+    created () {
+        if(this.price % 1 == 0) {
+            this.pricetext = this.price + ",-";
+        }
+    },
+    filters: {
+        descriptionLimit: (text) => {
+            if(text.length > 50) {
+                return text.substring(0, 50) + "...";
+            }
+            return text;
+        }
+    }
+}
+</script>
+<style scoped>
+    .dish-inner {
+        background-position: center center;
+        color: #33363a;
+        position: relative;
+        display: inline-block;
+        border-radius: 12px;
+        width: 100%;
+        height: 15vw;
+    }
+    .dish-inner.vertical .dish-label {
+        display: inline-block;
+        padding-top: 15px;
```

Comment:
Converge padding top and padding bottom into one line; padding: 15px 0;

Reply:
Resolved